tests/robot-tests/tests/general_public/notifications_absence.robot

```diff
@@ -43,5 +43,5 @@ Sign up for email alerts
     press keys  css:#subscriptionForm-email  mark@hiveit.co.uk
     user clicks button   Subscribe
 
-    user waits until page contains heading   Subscribed
+    user waits until page contains heading   Subscribed     120
```

Comment:
This doesn't seem like an area of the application that we should expect to be waiting around for 2 minutes

Reply:
I've seen this take a long time - I guess when the function is spinning up from cold.